src/server/services/pickup-queue/socket-methods.js

```diff
@@ -25,6 +25,31 @@ function queueWithoutPlayer(queue, playerId) {
 }
 
 /**
+ * Checks if players is blocked for join.
+ *
+ * @param {Object} app - The feathers app object.
+ * @param {String} userId - Player's ID.
+ * @returns {Boolean} - Is player blocked.
+ */
+async function checkBlock(app, userId) {
+  const query = { status: { $not: /game-finished|server-configuration-error/ } };
```

Comment:
Can we use an $or statement? This will make it easier to understand which pickups we get and if we add a status at some later point, we don't need to change this line

Reply:
tried, and $or does not work with strings apparently.